src/context/TreeContext.tsx

```diff
@@ -1,22 +1,18 @@
 import * as React from "react";
-
-interface TreeNode {
-  name: string;
-  props: Array<PropFormValue>;
-  state: Array<StateFormValue>;
-  parent: string;
-  children?: Array<TreeNode>;
-}
+import { findNodeAndInsert } from "../utils/tree/nodeHelpers";
```

Comment:
Would importing file in this way works

_**utils/tree/nodeHelpers**_

Reply:
Yes seems to work, great!